configure

```diff
@@ -1129,6 +1129,30 @@ EOF
     fi
 }
 
+
+check_acle_intrinsics() {
+    # Check whether -march=armv8-a+crc works correctly
+    cat > $test.c << EOF
+int main() { return 0; }
+EOF
+    if try $CC -c $CFLAGS -march=armv8-a+crc $test.c; then
+        ACLE_AVAILABLE=1
+        echo "Check whether -march=armv8-a+crc works ... Yes." | tee -a configure.log
```

Comment:
Consider using same language as `check_neon_intrinsics`..
```
Check whether -march=armv8-a+crc is available ... Yes."
```

Reply:
It is available, but unusable... It's matter of two compiler flags being mutually exclusive...

Reply:
Ok.